app/src/main/res/layout/alertdialog_sync_provider_chooser.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+
+    <ImageView
+        android:id="@+id/icon"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:paddingLeft="10dip"
```

Comment:
10dip is a pretty uncommon size for material design. Please use multiples of 4 (usually even multiples of 8).

Reply:
I switched all 10dip to 8dip.